### PR TITLE
feat(new-design): desktop version new-design

### PR DESCRIPTION
**Pull request by Shaharyar-saleem**

### CODE REVIEW

File: assets/fonts/Archivo/OFL.txt [hunk 1]
```diff
@@ -0,0 +1,93 @@
+Copyright 2016 The Archivo Project Authors (omnibus.type@gmail.com)
```
Comment (bobachef) [inline assets/fonts/Archivo/OFL.txt:1]: remove this file

File: index.html [hunk 2]
```diff
@@ -5,639 +5,740 @@
...
+          <p class="text-center breakdown-disc">After each buyback, transaction fees FOR SELLS are doubled for 30<br>
+            minutes, descending to the original rate with each minute</p>
         </div>
-        <div id="collapseThree" class="panel-collapse collapse">
-          <div class="panel-body">
-            Pazzive is one of the most advanced hyperdeflationary models crypto has ever seen. Implementing innovations
-            such as the Long Whale and the Anti-Whale as well as having the most efficient network fee to guarantee
-            maximum appreciation overtime. Pazzive is a crypto asset that will truly enable passive income for years to
-            come. It’s the 3rd official token of the Fuzion Super Blockchain which will boast an impressive 1 million
-            TPS at virtually no cost.
```
Comment (bobachef) [inline index.html:513]: We need to make sure that this is also added.

**Review by bobachef — CHANGES_REQUESTED** [review]
1. Please remove unnecessary files. 
2. Lets not lose any content that we had previously
3. make sure that all the links in the nav make sense

File: assets/fonts/Archivo/OFL.txt (hunk 1 above)
Comment (Shaharyar-saleem) [reply assets/fonts/Archivo/OFL.txt:1]: removed

File: index.html (hunk 2 above)
Comment (Shaharyar-saleem) [reply index.html:513]: FAQ  has been copied from previous version of web

Comment (Shaharyar-saleem) [reply index.html:513]: all above mentioned suggestions have followed and pushed.

**Pull request merged**